bbp/modules/default.nix

```diff
@@ -2392,7 +2414,7 @@ let
                 morphscale
                 morphsyn
                 muk
-                placement-algorithm
+                pybinreports
```

Comment:
We were planning to bury 'pybinreports' soon (to use Brion exclusively for binary report access).
Please let me know why it needs to be distributed, especially as a module.
Thanks.

Reply:
`git blame` tells me that @adevress added this in 3f2d8759c81a38dff47da2375e1f5877a5fabb50

Reply:
Hi Arseny,

pybinreports has been added rather recently for the need of Christian and its simplification workflow (with bglibpy) . I have nothing against removing it if you can guarantee me that Christian scripts do not need it anymore :)

Adrien

Reply:
OK, I'll check it with Christian then, just wanted to know where this one comes from.
You mean Christian O'Reilly, right?